dug_3_osfv.md

```diff
@@ -0,0 +1,102 @@
+class: center, middle, intro
+
+# &#x1F386; Dasharo Open Source Firmware Validation &#x1F386;
+
+.center[<img src="/remark-templates/dasharo-presentation-template/images/dasharo-sygnet-white.svg" width="150px" style="margin-left:-20px">]
+
+???
+
+---
+
+# Agenda
+
+* Introduction to Dasharo Open Sourve Firmware Validation (OSFV)
+* Exploring OSFV v0.2
+* Upcoming Features: OSFV v0.3 and Beyond
+* Q&A
+
+???
+
+---
+
+# History
+
+.center.image-99[![](/img/osfv.png)]
+
+* We use OSFV at least since 2018, when we started validate PC Engines releases
+* Using those scripts we over **50k** tests publicly relasing **150+** binaries
```

Comment:
we ... ?

Reply:
releasing

Spellchecker may be rquired

Reply:
I cannot afford your rates for spell checking to quote the guru, just leave one comment. I will fix those if/when I will find time before DUG